.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
I assume vscode changed this automatically for you. 

It's fine but we need to be mindful that fix on save will only work for vscode 1.85+ 

I have 1.85 and you should as well if vscode changed this for you